server/views/resources/shop.jade

```diff
@@ -3,14 +3,15 @@ block content
     h1.text-center Shop
     hr
     .row
-        .col-xs-12.col-sm-10.col-sm-offset-1
-            .row
-                .col-xs-12.text-center
-                    a(href='https://teespring.com/free-code-camp-feb' target='_blank')
-                        img.img-responsive.img-center(src='http://i.imgur.com/e50zjNy.jpg')
-                        h3 Free Code Camp "function call" logo t-shirt
-                    h4 100% Cotton. Unisex. $20 shipped. 
-                    h4 Only available until February 15.
-                    h5 Are you outside the US? Ships from the EU &thinsp;
-                        a(href='https://teespring.com/free-code-camp-t-shirt-eu-shop' target='_blank') here
-                        | .
\ No newline at end of file
+        .col-xs-12.col-sm-8.col-sm-offset-2.col-md-6.col-md-offset-3.text-center
+            img.img-responsive.img-center(src='http://i.imgur.com/MH1CvwY.jpg')
```

Comment:
Also, I found it hard to distinguish the scale of the stickers.  I know they're 2", but it's hard to tell from the image.  They may actually be larger on my screen than in real life.

Banana for scale?